testsuite/integration/web/web-enable-elytron.cli

```diff
@@ -1,6 +1,6 @@
 embed-server --server-config=standalone.xml
 
-/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
+#/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
```

Comment:
This line ensures, that the Undertow **default security domain** references Elytron instead of legacy security.